utils/vaultsigner/vaultsigner.go

```diff
@@ -159,3 +161,32 @@ func New(client *api.Client, name string) (*VaultSigner, error) {
 
 	return &VaultSigner{Client: client, KeyName: name, KeyVersion: 1}, nil
 }
+
+func Connect() (client *api.Client, err error) {
```

Comment:
can you add a comment for this function?
see the other exported functions for the format - e.g. `Connect ...`